src/app/login/page.tsx

```
@@ -0,0 +1,62 @@
+        </p>
+
+        {!newUser && <OAuthButtons />}
+      </div>
+      {!newUser && (
+        <>
+          <div className="flex w-xs items-center gap-6">
+            <Separator className="shrink border-2 border-lime-500" />
+            or
+            <Separator className="shrink border-2 border-lime-500" />
+          </div>
+        </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)